spec/acceptance/nodesets/debian-610-x64.yml

```diff
@@ -0,0 +1,12 @@
+HOSTS:
+  debian-610-x64:
+    roles:
+      - master
+    platform: debian-6-amd64
+    box: puppetlabs/debian-6.0.10-64-nocm
+    box_url: https://vagrantcloud.com/puppetlabs/boxes/debian-6.0.10-64-nocm
```

Comment:
that's fine by me, I don't mind having it there. Just wanted to make sure we weren't implicitly committing to support debian 6.